backend/core/api/congregation/dtos.go

```diff
@@ -3,15 +3,15 @@ package congregation
 import "backend/core/db/models"
 
 type VerifyCongregationPhoneDTO struct {
-	UserCode     string              `json:"userCode"`
-	Congregation models.Congregation `json:"congregation"`
+	UserCode     string              `json:"userCode" validate:"required"`
+	Congregation models.Congregation `json:"congregation" validate:"required"`
 }
 
 type SendCongregationVerificationCodeDTO struct {
-	PhoneNumber  string              `json:"phoneNumber"`
-	Congregation models.Congregation `json:"congregation"`
+	PhoneNumber  string              `json:"phoneNumber" validate:"required"`
```

Comment:
I haven't looked at go validate at all but I wonder if it has a built in validation for phone numbers. I guess we don't have to worry too much about it since this data is basically coming from the JW website so I'm assuming it's already validated on their side

Reply:
Yeah that's valid. JW will likely have their phone numbers validated since I believe they will have to verify it etc. when they register a new congregation. If we get into validating for new regular users, it could be useful so I'll look into it.